agent/api/task/service_connect.go

```diff
@@ -28,6 +28,10 @@ type ServiceConnectConfig struct {
 type RuntimeConfig struct {
 	// Host path for the administration socket
 	AdminSocketPath string `json:"adminSocketPath"`
+	// HTTP Path + Params to get statistical information
+	StatsRequest string `json:"statsRequest"`
+	// HTTP Path + Params to drain ServiceConnect connections
+	DrainRequest string `json:"drainRequest"`
```

Comment:
Currently, the control plane model has these variables removed per earlier discussions. Will add them back.

Reply:
I don't follow... why would you be adding them back? This isn't a ACS populated field

Reply:
I thought this is from SC properties. Looks like this is a SC config file. You can ignore this.